PETWorks/autoturn.py

```diff
@@ -230,6 +230,7 @@ def calculateThresholds(
         "k": [],
         "d": [],
         "t": [],
+        "lLimit": [],
```

Comment:
I suggest changing it to `l` to keep the naming strategy consistent with `k` because both denote the minimal measures of the dataset. What do you think?